.circleci/config.yml

```diff
@@ -116,12 +119,23 @@ commands:
                 name: "Start test relay server"
                 background: true
                 command: bash scripts/run-relay-server.sh
+      - run:
+          name: Install xvfb
+          command: |
+            sudo apt update
+            sudo apt install xvfb
+            sudo apt-get install -y libgbm-dev
+      - run:
+          name: Install cypress
+          command: ./node_modules/.bin/cypress install
       - run:
           name: Running integration tests
-          command: bash scripts/run-integration-tests.sh
+          command: bash scripts/run-integration-tests.sh <<parameters.browser>>
       - run:
           name: Upload test results to Datadog
           command: ./node_modules/.bin/datadog-ci junit upload --service <<parameters.datadog-test-service>> reports/junit-report.xml
+      - store_artifacts:
+          path: /home/circleci/project/cypress/screenshots
```

Comment:
This is cool, very nice. 😎

Reply:
Are these organized by browsers, channel (stable, beta, dev)? Any chance  these will get overwritten by other tests running in parallel?